### ISSUE TITLE
How to get `only_fields = ((),)` behavior? Goal: only shows fields declared in object type.

### ISSUE THREAD

**Issue by tony** [graphql-python/graphene-django#864]
First: Apologies for this if it's not descriptive or covered by graphene-django. I actually think this is something that's requested in other issues, via #671 #97

When updating graphene-django from 2.4.0 -> 2.8.0, behavior with `only_fields = ((), )` changed

I inherited this as a code snippet and don't know where it's from.

What it used to do: Only fields that were in defined in the graphene object type would be shown.

Is there a way to get the equivalent to that?

Also, is there a way to get computed fields to also show via `fields = ()`?

Assume this example:

```python
from django.db import models

class Reporter(models.Model):
    first_name = models.CharField(max_length=30)
    last_name = models.CharField(max_length=30)
    email = models.EmailField()

    def website(self):
        return "https://www.google.com"
```

Before: Only showed model fields declared in object type
Now: 

`fields = ((),)` gives:

```
  File "lib/python3.7/site-packages/graphene/utils/subclass_with_meta.py", line 52, in __init_subclass__
    super_class.__init_subclass_with_meta__(**options)
  File "lib/python3.7/site-packages/graphene_django/types.py", line 174, in __init_subclass_with_meta__
    construct_fields(model, registry, fields, exclude, convert_choices_to_enum),
  File "lib/python3.7/site-packages/graphene_django/types.py", line 42, in construct_fields
    if hasattr(model, name):
```

Changing to `fields = []` will show all fields on model.

The problem with this is its very exhaustive maintain a blacklist of fields via `exclude`. Since the model lives elsewhere in the code, it's thought of a bit separately. (Aside: This issue also exists in `django.contrib.admin` itself.)

```python
class Reporter(DjangoObjectType):
    first_name = graphene.String(required=True, description='First name')
    class Meta:
        model = ReporterModel
        fields = []

# expected
fields = list(Reporter._meta.fields.keys())
assert fields == ["first_name"]
```

Before: Allowed computed fields
Now: `Exception: "my_computed field" exists on model <class 'mymodel.Reporter'> but it's not a field.`

```python
class Reporter(DjangoObjectType):
    website = graphene.String(required=True, description='First name')
    class Meta:
        model = ReporterModel
        fields = ['website']

# Expected
fields = list(Reporter._meta.fields.keys())
assert fields == ["website"]
```

**Comment by jkimbo** [comment]
@tony if you don't want any fields from the Django model then why are you using a `DjangoObjectType` and not just a plain `ObjectType`?

Also the issue with computed types should be resolved in #862 which I'm hoping to release as v2.8.1 soon.

**Comment by tony** [comment]
@jkimbo 

> @tony if you don't want any fields from the Django model then why are you using a DjangoObjectType and not just a plain ObjectType?

In practice, I use fields from the django model. The point is I only want to "opt-in" to fields in the Django models declared in the object type.

Does that make sense?

> Also the issue with computed types should be resolved in #862 which I'm hoping to release as v2.8.1 soon.

That will be good! In that case, when v2.8.1 is released I can update the original issue that it's resolved for that aspect. (Assuming this issue isn't closed as a `wontfix` :laughing: )
